### PR TITLE
Added a reminder in slack notifications to run paasta status if it takes too long

### PR DESCRIPTION
**Pull request by solarkennedy**
This time I'm not using the sticht timer mechanism.

### CODE REVIEW

File: paasta_tools/cli/cmds/mark_for_deployment.py [hunk 1]
```diff
@@ -799,6 +851,7 @@ def on_enter_deploying(self):
                 target=self.do_wait_for_deployment, args=(self.commit,), daemon=True
             )
             thread.start()
+            self.schedule_paasta_status_reminder()
 
     def on_exit_deploying(self):
         self.wait_for_deployment_green_lights[self.commit].clear()
```
Comment (EvanKrall) [inline paasta_tools/cli/cmds/mark_for_deployment.py]: you could throw some code to cancel the reminder here

Comment (solarkennedy) [reply paasta_tools/cli/cmds/mark_for_deployment.py]: In order to do that though, I would have to build up the same kind of `timer_handle` logic that stitch does, right?

Comment (EvanKrall) [reply paasta_tools/cli/cmds/mark_for_deployment.py]: yeah, youd have to save the return value of call_later.

Comment (solarkennedy) [reply paasta_tools/cli/cmds/mark_for_deployment.py]: ok. I'm just in general hesitant to turn this functionality into its own full state machine

Comment (solarkennedy) [reply paasta_tools/cli/cmds/mark_for_deployment.py]: Ok. Cancelation support added.

**Pull request merged**